README.md

```diff
@@ -1,65 +1,62 @@
 
-## Install and usage
+### 2.4 TL;DR
+Does this plugin replace the need to think about structure? No. But it could relief you of the tedious work that has to happen when you just want to allocate files to a broad category and, what's even bigger, it will relief you of the pain to manually go through all the files and change their "parent-category whenever a topic gets too big or you want to move it somewhere else. Basically all you have to do is save things where they belong and the plugin will map that basic structure out for you. You can then, on top of that, add whatever MOC, index or tag logic you like.
 
-Similarly to any other plugins it is downloadable within Obsidian. Then, after enabling it, you will be able to configure Zoottelkeeper in its config interface. The following options can be configured:
+## 3. Installation and Settings
+Similarly to any other plugins it is downloadable within Obsidian. Then, after enabling it, you will be able to configure Zoottelkeeper in its config interface.
 
-- **The prefix of the index files**. Please note that it must be unique, othervise, normal notes with the same note name (with the parent folder name at its end, check the example)will be recognized as index files, and in this cases they will be updated! 
+![image](https://user-images.githubusercontent.com/46029522/126864195-4a8c7dd6-54ca-435e-a0bf-5a6520083609.png)
 
-- **Style of the index items**. Index items can be generated as
-  - pure Obsidian links, 
-  - list items (dots)
-  - links with checkboxes
+### 3.1 Clean Files
+There is a toggle to clean the indexed files. Depending on whether you would like to see the full file-path or not.
 
+### 3.2 Choose your List-Style
+There is three different types of lists for you to choose from:
+	- pure Obsidian links, 
+	- list items (dots)
+	- links with checkboxes
 
+### 3.3 Choose your Index Prefix
+Depending on your preferences, you can set any prefix to your index-files (or none at all). (Please note that the prefix must be unique, otherwise, normal notes with the same note name might be recognized as index files, and in this cases they will be updated! 
 
-## Tip for Graph view
+### 3.4 Enable Meta Tags
+You can choose to add YAML Meta Tags to your automatically generated index-files.
 
-As the folder links don't point to real notes (because they are folders), to make them visible in the Graph view, please enable the option of viewing orphan notes.
+### 3.5 Set Custom Meta Tags
+You can set one or multiple custom Meta Tags. Since they are displayed in the YAML format, you don't need to add a '#'.
+If you're setting multiple tags please make sure to separate them with commas.
 
+### 3.6 Additional Things
+- The indexed files are sorted alphabetically.
+- The file and the folder are no longer listed in the in the index-file. (This currently only works if you toggle 'Clean Files'.)

```

Comment:
I think with a solution in the other PR this feat. is going to be supported by default.